airflow/providers/amazon/aws/operators/glue_crawler.py

```diff
@@ -81,8 +89,22 @@ def execute(self, context: Context):
 
         self.log.info("Triggering AWS Glue Crawler")
         self.hook.start_crawler(crawler_name)
-        if self.wait_for_completion:
+        if self.deferrable:
+            self.defer(
+                trigger=GlueCrawlerCompleteTrigger(
+                    crawler_name=crawler_name,
+                    poll_interval=self.poll_interval,
+                    aws_conn_id=self.aws_conn_id,
+                ),
+                method_name="execute_complete",
+            )
+        elif self.wait_for_completion:
             self.log.info("Waiting for AWS Glue Crawler")
             self.hook.wait_for_crawler_completion(crawler_name=crawler_name, poll_interval=self.poll_interval)
 
```

Comment:
Does this wait for completion even if `wait_for_completion` is false?

Reply:
Oh sorry, disregard this comment. I read the indentation wrong when I was reviewing it.